src/music/sound.py

```diff
@@ -1,21 +1,46 @@
+SYMBOLS = ['C', 'C#', 'D', 'D#', 'E',
+           'F', 'F#', 'G', 'G#', 'A', 'A#', 'B']
+REVERSE_SYMBOLS = {
+    s: i for i, s in enumerate(SYMBOLS)
+}
+REVERSE_SYMBOLS.update(
+    {
+        s.lower(): i for i, s in enumerate(SYMBOLS)
+    }
+)
+REVERSE_SYMBOLS.update(
+    {
+        'cis': 1,
+        'dis': 3,
+        'fis': 6,
+        'gis': 8,
+        'ais': 10
+    }
+)
+
+
 class Sound:
-    def __init__(self, note, timestamp=None, duration=None):
+    def __init__(self, note=None, timestamp=None, duration=None,
+                 duration_signature=None, symbol=None):
         self.note = note
+        if symbol is not None:
+            self.note = REVERSE_SYMBOLS[symbol]
         self.timestamp = timestamp
         self.duration = duration
+        self.duration_signature = duration_signature
+        self.beat_fraction = None
```

Comment:
Should not be a property - it is used only locally in places there it's used, should be deduced from metrum

Reply:
as Kuba mentioned above, should use only rhytmic_value

Reply:
Fixed